src/step1-index.js

```diff
@@ -1,4 +1,37 @@
-/**
- * step 1의 시작점이 되는 파일입니다.
- * 브라우저 환경에서 사용하는 css 파일 등을 불러올 경우 정상적으로 빌드할 수 없습니다.
- */
+import { readLineAsync } from './utils/readlineAsync.js'
+import { getTicket, LOTTO_PRICE } from './lotto.js'
+import { calculateProfitRate, getRank, getRankStatistics } from './lottoGame.js'
+import { printProfit, printResults, printTicket } from './lottoPrint.js'
+
+async function start() {
+  const tickets = await getTickets()
+  printTicket(tickets)
+
+  const { winningNumbers, winningBonusNumber } = await getWinningNumber()
+  const { stats, totalWinnings } = getStatistics(tickets, winningNumbers, winningBonusNumber)
+  printResults(stats, totalWinnings)
+
+  const profitRate = calculateProfitRate(tickets.length * LOTTO_PRICE, totalWinnings)
+  printProfit(profitRate)
+}
+
+async function getTickets() {
+  const purchaseAmount = await readLineAsync('구입 금액을 입력해 주세요. ')
+  return getTicket(purchaseAmount)
+}
+
+async function getWinningNumber() {
+  let winningNumbers = await readLineAsync('당첨 번호를 입력해 주세요. ')
+  winningNumbers = winningNumbers.split(',').map(num => Number(num.trim()))
```

Comment:
사용자가 입력한 값과, 사용자의 입력 값을 토대로 winingNumber를 추출한 값을 따로 따로 선언해 준다면 보다 분명해지고 `let` 키워드의 사용 빈도도 줄어들 수 있을 거예요 :) 

다만, 도메인이 복잡하지 않기도 하고 살려뒀다고 해서 고쳐야 하는 코드라고 생각지는 않습니다!

```suggestion
  const inputWinningNumbers = await readLineAsync('당첨 번호를 입력해 주세요. ')
  const winningNumbers = winningNumbers.split(',').map(num => Number(num.trim()))
```